firefox-ios/Client/Telemetry/GleanUsageReporting.swift

```diff
@@ -42,7 +42,7 @@ class GleanUsageReporting: GleanUsageReportingApi {
 class GleanLifecycleObserver {
     private let gleanUsageReportingApi: GleanUsageReportingApi
     private var id: TimerId?
-    private var isObserving: Bool = false
```

Comment:
this seems unrelated? intentional?

Reply:
Swiftlint fix :(. I couldn't push without it